Ryujinx.HLE/HOS/Applets/SoftwareKeyboard/SoftwareKeyboardApplet.cs

```diff
@@ -76,6 +88,28 @@ private void Execute()
                 _keyboardConfig.StringLengthMax = 100;
             }
 
+            var args = new SoftwareKeyboardUiArgs
+            {
+                HeaderText = _keyboardConfig.HeaderText,
+                SubtitleText = _keyboardConfig.SubtitleText,
+                GuideText = _keyboardConfig.GuideText,
+                SubmitText = (!string.IsNullOrWhiteSpace(_keyboardConfig.SubmitText) ? _keyboardConfig.SubmitText : "OK"),
+                StringLengthMin = _keyboardConfig.StringLengthMin, 
+                StringLengthMax = _keyboardConfig.StringLengthMax,
+                InitialText = initialText ?? ""
+            };
+
+            // Call the configured GUI handler to get user's input
+            try
+            {
+                _textValue = _device.UiHandler.DisplayInputDialog(args);
+            }
+            catch(Exception e)
```

Comment:
```suggestion
            catch (Exception e)
```
Sorry for nitpicking.